qa/tasks/cephadm.py

```diff
@@ -447,6 +447,61 @@ def pull_image(ctx, config):
     finally:
         pass
 
+@contextlib.contextmanager
+def setup_ca_signed_keys(ctx, config):
+    # generate our ca key
+    cluster_name = config['cluster']
+    bootstrap_remote = ctx.ceph[cluster_name].bootstrap_remote
+    bootstrap_remote.run(args=[
+        'ssh-keygen -t rsa -f /root/ca-key -N ""'
```

Comment:
```
2023-07-08T22:15:07.219 DEBUG:teuthology.orchestra.run.smithi060:> 'ssh-keygen -t rsa -f /root/ca-key -N ""'
2023-07-08T22:15:07.243 DEBUG:teuthology.orchestra.run:got remote process result: 127
2023-07-08T22:15:07.244 INFO:teuthology.orchestra.run.smithi060.stderr:bash: ssh-keygen -t rsa -f /root/ca-key -N "": No such file or directory
```
https://pulpito.ceph.com/adking-2023-07-08_17:43:37-orch:cephadm-wip-adk4-testing-2023-07-08-1112-distro-default-smithi/7330185/

Reply:
```
[root@smithi196 ~]# ssh-keygen -t rsa -f /root/ca-key -N ""
Generating public/private rsa key pair.
Your identification has been saved in /root/ca-key.
Your public key has been saved in /root/ca-key.pub.
The key fingerprint is:
SHA256:En57JLjP95p139go3ft3Bc3hv0vXX7ODY849VA75SWc root@smithi196
The key's randomart image is:
+---[RSA 3072]----+
|                 |
|               . |
|      .       .+.|
|     . o      +oE|
|      + S .   .*=|
|       + +     o*|
|      . . . ..++*|
|       o ..oo=+O@|
|        o.oo+++=#|
+----[SHA256]-----+
[root@smithi196 ~]# ls /root/
anaconda-ks.cfg  ca-key  ca-key.pub  cobbler.ks  ks-post.log  ks-post-nochroot.log  ks-pre.log  original-ks.cfg
```
works when run directly on the node after the test fails

Reply:
appears to be an issue with the quoting around the command
```
[root@smithi196 ~]# 'ssh-keygen -t rsa -f /root/ca-key -N ""'
-bash: ssh-keygen -t rsa -f /root/ca-key -N "": No such file or directory
```

Reply:
```suggestion
        'ssh-keygen', '-t', 'rsa', '-f', '/root/ca-key', '-N', '""''
```